src/bundle/Resources/views/themes/admin/ui/component/tag_view_select.html.twig

```diff
@@ -0,0 +1,58 @@
+{% set attr = attr|default({})|merge({
+    class: 'ibexa-tag-view-select__btn-select-path btn ibexa-btn ibexa-btn--secondary ' ~ attr.class|default(''),
+}) %}
+{% set separator = separator|default(',') %}
+{% set values = form.vars.value|split(separator)|filter(value => value is not empty) %}
+{% set has_value = values|length > 0 %}
+{% set can_delete_last = can_delete_last is defined ? can_delete_last : false %}
+{% set single_select = single_select is defined ? single_select : false %}
+{% set is_delete_visible = can_delete_last or values|length != 1 %}
+{% set default_title = 'tag_view_select.info'|trans|desc('Select location') %}
+{% set is_disabled = attr.disabled is defined ? attr.disabled : false %}
+{% set default_type = 'tag_view_select.default_type'|trans|desc('path') %}
+{% set type = type|default(default_type) %}
+
+<div
+    class="ibexa-tag-view-select"
+    data-single-select="{{ single_select }}"
+    data-can-delete-last="{{ can_delete_last }}"
+    {% if is_disabled %}disabled{% endif %}
+>
+    <div class="ibexa-tag-view-select__info">
+        {{ title|default(default_title) }}
+    </div>
+    <div
+        class="ibexa-tag-view-select__selected_list"
+        data-template="{{ include('@ezdesign/ui/component/tag_view_select_selected_item.html.twig', {
+            id: '{{ id }}',
+            name: '{{ name }}',
+        })|e('html_attr') }}"
+    >
+        {% for item in items %}
+            {{ include('@ezdesign/ui/component/tag_view_select_selected_item.html.twig', {
+                id: item.id,
+                name: item.name,
+                can_delete: is_delete_visible,
+                is_disabled
+            }) }}
+        {% endfor %}
+    </div>
+    {{ form_widget(form) }}
+    <button
+        type="button"
+        {{ block('attributes', 'bootstrap_5_layout.html.twig') }}
```

Comment:
Hmm, why you didn't use `attributes` macro from `@ezdesign/ui/component/macros.html.twig`?